Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -679,6 +679,9 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
     bool shouldPaintUsingCompositeCopy() const { return m_shouldPaintUsingCompositeCopy; }
     void setShouldPaintUsingCompositeCopy(bool copy) { m_shouldPaintUsingCompositeCopy = copy; }
 
+    bool renderingSuppressed() const { return m_renderingSuppressed; }
+    virtual void setRenderingSuppressed(bool suppressed) { m_renderingSuppressed = suppressed; }
```

Comment:
`setRenderingIsSuppressed`

Reply:
This naming needs to make it clear that the state affects not just this layer, but all descendants. Maybe `setRenderingIsSuppressedIncludingDescendants()` or something.